modules/configuring-oidc-authentication.adoc

```diff
@@ -0,0 +1,236 @@
+
+By configuring Red Hat Single Sign-On on {productname}, you can create a seamless authentication integration between {productname} and other application platforms like {ocp}. 
+
+[id="configuring-red-hat-sso-using-config-tool"]
+=== Configuring the Red Hat Single Sign-On Operator for the {productname} Operator
+
+Use the following procedure to configure Red Hat Single Sign-On for the {productname} Operator on {ocp}. 
+
+.Prerequisites 
+
+* You have configured Red Hat Single Sign-On for OpenShift. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6/html-single/server_installation_and_configuration_guide/index#operator[Red Hat Single Sign-On Operator]. 
+* You have configured TLS/SSL for your {productname} deployment _and_ for Red Hat Single Sign-On. 
+* You have generated a single Certificate Authority (CA) and uploaded it to your Red Hat Single Sign-On Operator _and_ to your {productname} configuration.  
+* You are logged into your {ocp} cluster. 
+* You have installed the OpenShift CLI (`oc`). 
+
+.Procedure 
+
+. Upload a custom SSL/TLS certificate to your {ocp} deployment by running the following commands: 
+
+.. Base64 encode the content of a single TLS/SSL certificate by entering the following command:
++
+[source,terminal]
+----
+$ cat ssl.crt | base64 -w 0
+----
++
+.Example output
++
+[source,terminal]
+----
+LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lKQUtsYW9WNHMrb1BtTUEw
+[truncated]
+----
+
+.. Create a Secret called `quay-enterprise-cert-secret` by entering the following command: 
++
+[source,terminal]
+----
+$ oc create create secret generic quay-enterprise-cert-secret --from-file=ca.crt=<path_to_certificate_file>
+----
+
+.. Mount the secret into the `qe` pod in `/conf/stack/extra_ca_certs`:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  ssl.crt:  LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lKQUtsYW9WNHMrb1BtTUE[truncated]
+kind: Secret
+metadata:
+  name: quay-enterprise-cert-secret
+type: Opaque
+----
+
+.. Edit the `quay-enterprise-app` deployment to add the following mount path under `volumeMounts`. This can be done from the {ocp} console or by using `oc edit`.
++
+[source,yaml]
+----
+    - mountPath: /conf/stack/extra_ca_certs
+      name: certvolume
+----
+
+.. Under the `volumes` section, add the `quay-enterprise-cert-secret` as a volume. For example:
++
+[source,yaml]
+----
+    - name: certvolume
+      secret:
+        defaultMode: 420
+        secretName: quay-enterprise-cert-secret
+----
+
+.. Save the manifest. The `Quay` pods should be rescheduled automatically. 
```

Comment:
The purpose of line 36 to line 91 is to upload SSO  CA certificate to quay pod, right?
If we suppose quay is deployed by operator, modify deployment won't work as expected.  The operator will restore the update.  

I think the simplest way to do it is to upload CA certificate by quay-config-editor UI,  access quay-config-editor  endpoint by browser, upload CA certificate in UI, then reconfigure quay. 

```
$ oc get route -l quay-component=quay-config-route -n quay-enterprise
NAME                              HOST/PORT                                                                                           PATH   SERVICES                          PORT   TERMINATION     WILDCARD
quayregistry-quay-config-editor   quayregistry-quay-config-editor-quay-enterprise.apps.whu413az23.qe.azure.devcluster.openshift.com          quayregistry-quay-config-editor   http   edge/Redirect   None

```

Reply:
Done!